andela/mrm-mobile
# #159805378 [WIP] User can extend meeting time

### What does this PR do?
Allows the user to extend meeting time after checking in.

### Description of Task to be completed.
- [x] User can view countdown timer.
- [x] User can extend meeting by 10, 20 or 30 minutes.
- [ ] User cannot extend meeting if room is booked for the next session.

### How should this be manually tested?
1. Set up the project as shown in README.
2. Run the app on an emulator and navigate to the room activity.
3. Ensure you have a meeting that should be ongoing and click ``` check in ```.
4. Click the time you want to extend meeting by. ``` +10, +20, +30 ```


### What are the relevant pivotal tracker stories?
159805378
[A user can extend meeting time on the meeting room screen](https://www.pivotaltracker.com/story/show/159805378)

### Screenshots
![ezgif com-optimize](https://user-images.githubusercontent.com/33759757/44976310-8b68d700-af6d-11e8-8887-4cdd61cda473.gif)
